src/hyp3_sdk/hyp3.py

```diff
@@ -30,7 +30,8 @@ def __init__(
         api_url: str = PROD_API,
         username: str | None = None,
         password: str | None = None,
-        prompt: bool = False,
+        token: str | None = None,
+        prompt: str | None = None,
```

Comment:
We'll probably want to test with various combinations of `username`, `password`, `token`, and `prompt` parameters, as well as netrc vs. no-netrc. I haven't fully digested the Mattermost discussion re: desired precedence.